_includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md

```diff
@@ -0,0 +1,1198 @@
+* Password set and reset console commands for customers.
+* Double opt-in for customer registration.
+* OAuth
+* Improved password security
+* Session validation
+
+{% endinfo_block %}
+
+## Install feature core
+
+Follow the steps below to install the Customer Account Management feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME         | VERSION          | INTEGRATION GUIDE                                                                                                                    |
+|--------------|------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core | {{site.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html) |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/customer-account-management: "{{site.version}}" spryker/oauth-customer-connector:"^1.6.0" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following modules have been installed:
+
+| MODULE                  | EXPECTED DIRECTORY                           |
+|-------------------------|----------------------------------------------|
+| Customer                | vendor/spryker/customer                      |
+| CustomerGroup           | vendor/spryker/customer-group                |
+| CustomerNote            | vendor/spryker/customer-note                 |
+| CustomerNoteGui         | vendor/spryker/customer-note-gui             |
+| CustomerStorage         | vendor/spryker/customer-storage              |
+| CustomerValidationPage  | vendor/spryker-shop/customer-validation-page |
+| Oauth                   | vendor/spryker/oauth                         |
+| OauthCryptography       | vendor/spryker/oauth-cryptography            |
+| OauthCustomerConnector  | vendor/spryker/oauth-customer-connector      |
+| OauthCustomerValidation | vendor/spryker/oauth-customer-validation     |
+| OauthRevoke             | vendor/spryker/oauth-revoke                  |
+
+{% endinfo_block %}
+
+### 2) Set up configuration
+
+1. Set the required OAuth config:
+
+{% info_block infoBox %}
+
+For more details about key generation, see [OAuth 2.0 Server Installation](https://oauth2.thephpleague.com/installation/).
+
+{% endinfo_block %}
+
+**config/Shared/config_default.php**
+
+```php
+$config[OauthConstants::PRIVATE_KEY_PATH] = 'file://path/to/private.key';
+$config[OauthConstants::PUBLIC_KEY_PATH] = 'file://path/to/public.key';
+$config[OauthConstants::ENCRYPTION_KEY] = 'generated-encryption-key';
+$config[OauthConstants::OAUTH_CLIENT_IDENTIFIER] = 'client-identifier';
+$config[OauthConstants::OAUTH_CLIENT_SECRET] = 'client-secret';
+```
+
+2. Register publish and synchronization queue:
+
```

Comment:
I see `src/Pyz/Zed/Customer/Persistence/Propel/Schema/spy_customer.schema.xml` is adjusted at nonsplit PR but not mentioned